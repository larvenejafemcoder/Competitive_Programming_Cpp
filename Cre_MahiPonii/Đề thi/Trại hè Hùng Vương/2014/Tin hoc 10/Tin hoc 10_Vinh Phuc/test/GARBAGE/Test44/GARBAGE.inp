NDUSEDSUESDESNSUUDSWNNEE
EDDDSE
WDUUSWW
NUSDEWWEUNSUDDEENUNNSUESDDDDSUWWNNDUNDSWSSSSWEEEEWNUWWWEEDUWSUWUUNUEDW
ESUWDUUWUW
SUWDUWSSSEWDNSNUDEENSUUDUNUDNEEDNWDDDWNDDNSEWESWEWNDEDUNUWDNSDWNWWNEWSENNDDDWNSEWUSWEN
U 96
