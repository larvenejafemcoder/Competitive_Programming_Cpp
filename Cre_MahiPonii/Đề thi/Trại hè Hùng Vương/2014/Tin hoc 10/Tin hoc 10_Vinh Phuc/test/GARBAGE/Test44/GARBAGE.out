78134349824285344951047108382937027298983565672556526645992411521774521827838042945012736594906931761309836048588024520829036887403946698025471564
